examples/templates/logo-strip.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Logo strip
+category: _templates
+---
+
+<div class="row">
+    <div class="mobile-col-1 tablet-col-1 col-2 u-first-col--tablet u-first-col--desktop">
```

Comment:
This is extremely fiddly. Why not create a pattern that encapsulates all this single style classes?

Reply:
@anthonydillon 

> Why not create a pattern that encapsulates all this single style classes?

I've proposed this separately as "layout presets", but it isn't on the roadmap yet.
Common scenarios should indeed be invoked much easier. In this case, a layout preset could be: .layout--4-6-6 where the digits indicate the number of elements per row per breakpoint.

But lacking a set of presets as we do at the moment, we need flexibility. A logo strip on with a different number of logos might require 2-2-4 (2 logos on mobile, 2 on tablet, 4 on desktop).

What is common between those is the need to wrap logos in the same .row, so we don't get a gap. What will differ depending on the content is the exact number of columns required per breakpoint.